### PR TITLE
user-reset-password

### PR DESCRIPTION
**Pull request by Ezejiohan**
The user reset password hasn't been tested yet, but just check it

### CODE REVIEW

File: controllers/userC.js [hunk 1]
```diff
@@ -23,4 +23,27 @@ const createUser = asyncWrapper(async(req, res, next) => {
...
+    const updatePassword = await updateUser(req.params.id, {
+        password: hashPassword
+    });
+
+    await user.save();
+
+    res.status(200).json({ msg: "Password reset successful", updatePassword });
+});
+
+module.exports = { createUser, resetUserPassword }
```
Comment (hizikex) [inline controllers/userC.js]: Add an extra line

File: routes/users.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
 const express = require('express');
-const { createUser } = require('../controllers/userC');
+const { createUser, resetUserPassword } = require('../controllers/userC');
 
 const userRoute = express.Router();
 
 userRoute.post('/register', (createUser));
+userRoute.patch('/resetUserPassword/:id/:token', (resetUserPassword));
```
Comment (hizikex) [inline routes/users.js]:
```suggestion
userRoute.patch('/:id/:token', (resetUserPassword));
```

Your endpoint should not have a verb in it

**Review by hizikex — CHANGES_REQUESTED** [review]

File: controllers/userC.js (hunk 1 above)
Comment (Ezejiohan) [reply controllers/userC.js]: Okay noted

File: routes/users.js (hunk 2 above)
Comment (Ezejiohan) [reply routes/users.js]: Okay, noted

File: controllers/userC.js [hunk 3]
```diff
@@ -23,4 +23,29 @@ const createUser = asyncWrapper(async(req, res, next) => {
...
+    });
+
+    await user.save();
+
+    res.status(200).json({ msg: "Password reset successful", updatePassword });
+});
+
+module.exports = { createUser, 
+    resetUserPassword 
+}
```
Comment (hizikex) [inline controllers/userC.js]: Did you test this code? @Ezejiohan

**Pull request merged**